### DISCUSSION TITLE
Can this be used only as a client?

### DISCUSSION THREAD

**Discussion by AlejandroAkbal** [Hebilicious/authjs-nuxt#131] [Q&A] [answered]
My Nuxt is in static generation mode

You can view the repo [here](https://github.com/Rule-34/App/tree/feat/upgrade-to-nuxt-3) 

I want to authenticate with credentials (user and password) to a Nestjs backend
But from all the examples it seems like you use ``playgrounds/credentials/server/api/auth/[...].ts`` to set up the credentials handler, which is not available in static generation

How can I authenticate while using only the client?

**Comment by Hebilicious** [comment, marked as answer]
It can't, Auth.js works on the server.
If you want to authenticate to a NestJS backend, you don't need to use this module. You can look at the source code of the composable and implement something similar that is specific to your own backend. You could even extract it as your own internal module.
